nin/frontend/app/scripts/filters/errorExcerpt.js

```diff
@@ -1,5 +1,9 @@
 function errorExcerpt($sce) {
   return function(error, fileCache) {
+    if (fileCache === undefined || !error.path in fileCache) {
+      return;
```

Comment:
Our error handling code is actual spaghetti atm:
https://github.com/ninjadev/nin/blob/550b6726a45c739954e31c030c2928c329a0a282/nin/frontend/app/views/top-panel.html#L20

Three things are always printed in the error display:
- The first line of the errors
- The file excerpt where the errror came from (This is not always appliccable, as the error might not have a file source, as is the case when the error originates from our GUI for instance)
- The rest of the lines of the errors.

This makes it so that the file excerpt doesn't crash NIN while trying to show the original error,
while still showing the full error.